### ISSUE TITLE
Slides Not Inheriting App Theme

### ISSUE THREAD

**Issue by CodyEddings** [AppIntro/AppIntro#486]
Regardless of my declaration of a theme to use at either the application or activity level in my manifest file, the intro slides seem to ignore it and default to white text. Surely there has to be a way around this since some of the sample screenshots show non-white text and different fonts. 

**AppIntro Version**:4.2.2

**Device/Android Version**:Physical Galaxy S6 Edge/7.0 Nougat & Emulated Pixel/8.0 Oreo

**Issue details / Repro steps / Use case background**: I'm trying to simply change the color of the text my intro slides to match the theme in the rest of my app (eg. the main activity), and disable the activity title bar. I imagine this is possible but I'm not sure how to do so. Since we're not supposed to use setContentView() in our IntroActivity I've been unable to programatically set the theme in IntroActivity's onCreate(), and setting the theme of the activity in my manifest xml doesn't seem to make a difference in any intro slide's theme.

**Your Code**:  AndroidManifest.xml
```
<?xml version="1.0" encoding="utf-8"?>
<manifest 
	...>

    <application
        android:name="MyApplication"
        android:allowBackup="true"
        android:icon="@drawable/launcher"
        android:label="@string/app_name"
        android:theme="@style/GSMaterialTheme"> <!-- Theme set here-->

        <activity 								<!-- Theme applies successfully to this activity-->
            android:name=".MainActivity"
            android:configChanges="orientation|keyboardHidden|screenSize"
            android:screenOrientation="portrait">
			...
        </activity>

        <activity								<!--But doesn't apply to this one -->
            android:name=".FirstRunActivity"
            android:label="@string/app_intro"/>

		...
		
    </application>

</manifest>

```
styles.xml
```
<?xml version="1.0" encoding="utf-8"?>
<resources>
    <style name="GSMaterialTheme" parent="Theme.AppCompat">
        <item name="colorPrimary">#000000</item>
        <item name="colorPrimaryDark">#FFD600</item>
        <item name="colorAccent">#C4252C</item>
        <item name="android:textColorPrimary">#FFD600</item>
        <item name="preferenceTheme">@style/PreferenceFixTheme</item>
        <item name="android:activatedBackgroundIndicator">?android:attr/selectableItemBackground</item>
    </style>
</resources>
```

**Stack trace / LogCat**: No log messages to post
```ruby
paste stack trace and/or log here
```

**Comment by CodyEddings** [comment]
After more testing it appears that the activity is being set to the theme in the manifest, however the text color doesn't apply itself to the text color in said theme. How can I make the text on the intro slides change color?

**Comment by avluis** [comment]
@CodyEddings Are you creating your slides via a custom fragment implementation (making use of `SampleSlide`) or with the built-in `AppIntroFragment`?

**Comment by Anutha02** [comment]
@cortinico Fixed in #647
